src/main/java/frc/robot/subsystems/shooter/GetToTargetVelocity.java

```diff
@@ -23,21 +24,22 @@ public GetToTargetVelocity(ShooterSub shooter, Dashboard dashboard) {
   @Override
   public void initialize() {
 
-      SmartDashboard.putString("command status", "ramping up");
+      SmartDashboard.putString("shooter", "ramping up");
```

Comment:
i really dont understand why you did this. maybe im just dumb but does this mean that the command status will be the last command? and that the shooter tab will show ramping up? why not all in one place?